tests/test_python_memaddr.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+    test_python_memaddr
+    ~~~~~~~~~~~~~~~~~~~
+
+    Tests that memory addresses are not shown in python code documentation.
+
+    :copyright: Copyright 2007-2016 by the Sphinx team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from util import with_app
+
+
+@with_app('html', testroot='python-memaddr')
+def test_memory_address(app, status, warning):
+    app.builder.build_all()
+    for f in ['index.html','searchindex.js']:
+        t = (app.outdir / f).text()
+        refs = t.count("0x")
+        assert refs == 0, "A memory address was found %d times in the file %s." % (refs, f)
```

Comment:
Minor nit:  This makes it sound as if "a [single] memory address was found [multiple] times in the file."  Maybe "%d memory addresses were found in file %s", or "File %s contained %d memory addresses"?